src/main/index.ts

```diff
@@ -146,9 +158,19 @@ if (!isFirstInstance) {
   };
 
   app.on('session-created', (session) => {
+    let reservedStateValues = [];
     if (!isDevelopment) {
       log.info('Cleaning local storage on session creation');
-      session.clearStorageData({ storages: ['localstorage'] }).catch(log.error);
+
```

Comment:
Are you sure this code is working? When I stepped through the code, this piece is called before `mainWindow = newWindow;`, meaning `mainWindow != null` is inevitably false, so this code will never execute.

Record a GIF to demo a packaged app to be working on Windows and/or Linux and/or macOS and add that GIF to the PR description.